@xen-orchestra/web-core/lib/components/UiLegend.vue

```diff
@@ -0,0 +1,97 @@
+<!-- v1.0 -->
+<template>
+  <li class="legend">
+    <span class="circle" :class="circleColorClass"></span><span class="label typo p3-regular">{{ label }}</span
```

Comment:
- DS uses an icon as circle.
- Prefer using a slot instead of a prop for the main content.

```suggestion
    <UiIcon :icon="faCircle" class="circle-icon" />
    <span class="label typo p3-regular"><slot /></span>
```